modules/electrophysiology_browser/help/view_session.md

```diff
@@ -0,0 +1,15 @@
+# View Session
+
+This page displays all available information collected for an individual recording session.
```

Comment:
```suggestion
This page displays information from an individual recording session.
```

it's not actually true that "all available" information is displayed on this page.